src/Sylius/Behat/Page/Admin/Product/IndexPage.php

```diff
@@ -74,13 +89,29 @@ public function checkLastProductHasDataAttribute(string $attributeName): bool
         return $this->getElement('last_product')->find('css', sprintf('[%s]', $attributeName)) !== null;
     }
 
+    public function isEnabledFilterApplied(): bool
+    {
+        return $this->getElement('enabled_filter')->getValue() === 'true';
+    }
+
+    public function checkPageNumber(): int
+    {
+        /** @var NodeElement $element */
+        $element = $this->getElement('pagination_buttons');
+        $buttons = $element->find('css', '.active');
+
+        return (int) $buttons->getText();
```

Comment:
```suggestion
        $buttons = $this->getElement('pagination_buttons');
        $activeButton = $buttons->find('css', '.active');

        return (int) $activeButton->getText();
```
or just inline the return
```suggestion
        $buttons = $this->getElement('pagination_buttons');

        return (int) $buttons->find('css', '.active')->getText();
```